container/image.go

```diff
@@ -66,7 +67,7 @@ func (ug *imageURLGetter) CACert() []byte {
 
 // ImageDownloadURL determines the public URL which can be used to obtain an
 // image blob with the specified parameters.
-func ImageDownloadURL(kind instance.ContainerType, series, arch, cloudimgBaseUrl string) (string, error) {
+func ImageDownloadURL(kind instance.ContainerType, series, arch, cloudimgBaseUrl, cloudimgStream string) (string, error) {
```

Comment:
please put the stream param after arch

func ImageDownloadURL(kind instance.ContainerType, series, arch, cloudimgStream, cloudimgBaseUrl string) (string, error) {